R/git.R

```diff
@@ -69,11 +69,14 @@ git_checkout_branch <- function(name, force = FALSE, root = NULL,
 
 git_clone_local <- function(source, destination = NULL) {
   orderly::orderly_log("git", "clone")
+  remote_url <- git_run(c("config", "--get", "remote.origin.url"),
+                        root = source, check = TRUE)
   if (!length(destination)) {
     destination <- tempfile()
     dir_create(destination)
   }
-  git_run(c("clone", source, destination), check = TRUE)
+  git_run(c("clone", remote_url$output, destination),
```

Comment:
I am not sure this is what we want to do, as this will be much slower than cloning the repo from the local source, plus we need some longer term way of keeping these in sync.

So we have 3 different places:

1. our remote
2. our main local copy
3. our "alternative" root

We do the clone at the beginning, at which point 3 will exactly match the state of 1 or 2

However, if more commits are added to the remote then 1 and 2 are out of sync. If we update 2 to match 3 then 3 is out of sync. With your rewiring, then 2 and 3 never get synced up?

I wonder if what we should do is:

1. local clone from 2 -> 3, but adding argument `--origin=local`
2. do `git remote add origin remote_url && git fetch origin` on the alternative remote

Then we can more easily update to refresh the position of the tree, but that logic is going to need adding somewhere